Formula/gerbil-scheme.rb

```diff
@@ -3,7 +3,8 @@ class GerbilScheme < Formula
   homepage "https://cons.io"
   url "https://github.com/vyzo/gerbil/archive/v0.17.tar.gz"
   sha256 "1e81265aba7e9022432649eb26b2e5c85a2bb631a315e4fa840b14cf336b2483"
-  license "Apache-2.0"
+  license all_of: ["LGPL-2.1-or-later", "Apache-2.0"]
```

Comment:
This actually should be `any_of:` (as it is dual licensed)

addressed in https://github.com/Homebrew/homebrew-core/pull/122796